tests/multus_connectivity_test.go

```diff
@@ -0,0 +1,223 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+const (
+	ovsPodYaml         = "tests/manifests/ovs-vsctl-pod.yml"
+	ovsVlanNet         = "tests/manifests/ovs-vlan-net.yml"
+	ovsVsCtl           = "ovs-vsctl"
+	bridgeName         = "br1"
+	ipAddCmd           = "sudo ip add a %s/24 dev %s \n"
+	ipUpCmd            = "sudo ip link set up %s \n"
+	privilegedTestUser = "privileged-test-user"
+)
+
+type NodesToIp struct {
+	name string
+	ip   string
+}
+
+var _ = FDescribe("Multus Network Connectivity", func() {
+	flag.Parse()
+	virtClient, err := kubecli.GetKubevirtClient()
+	ktests.PanicOnError(err)
+
+	var pods *corev1.PodList
+	var vmiList [2]*v1.VirtualMachineInstance
+	var ovsVmsIp [2]string
+	var nodesNames [2]string
+	var defaultVmsIp [2]string
+	var nodesToIpList [2]NodesToIp
+	privilegedTestUser := "privileged-test-user"
+
+	BeforeEach(func() {
+		ovsVmsIp = [2]string{"192.168.0.1", "192.168.0.2"}
+		_, _, err := ktests.RunCommand("oc", "create", "serviceaccount", privilegedTestUser)
```

Comment:
You can reuse the test_admin that the other tests are using. It's details can be found in util.go.

Reply:
What do you mean? 
I need to run a privileged pod on the test namespace.
Can I use test_admin for 
serviceAccount and serviceAccountName in the pod yaml?